src/main/resources/db/migration/v0.30.0.1__linkml_v2.0.0_update.sql

```diff
@@ -0,0 +1,2574 @@
+	FROM synonym;
+	
+INSERT INTO auditedobject (tablename, id, old_id, createdby_id, updatedby_id, datecreated, dateupdated, dbdatecreated, dbdateupdated, internal, obsolete)
+	SELECT 'vocabulary', nextval('auditedobject_seq'), id, createdby_id, updatedby_id, datecreated, dateupdated, dbdatecreated, dbdateupdated, internal, obsolete
+	FROM vocabulary;
+	
+INSERT INTO auditedobject (tablename, id, old_id, createdby_id, updatedby_id, datecreated, dateupdated, dbdatecreated, dbdateupdated, internal, obsolete)
+	SELECT 'vocabularyterm', nextval('auditedobject_seq'), id, createdby_id, updatedby_id, datecreated, dateupdated, dbdatecreated, dbdateupdated, internal, obsolete
+	FROM vocabularyterm;
+	
+INSERT INTO auditedobject (tablename, id, old_id, createdby_id, updatedby_id, datecreated, dateupdated, dbdatecreated, dbdateupdated, internal, obsolete)
+	SELECT 'vocabularytermset', nextval('auditedobject_seq'), id, createdby_id, updatedby_id, datecreated, dateupdated, dbdatecreated, dbdateupdated, internal, obsolete
+	FROM vocabularytermset;
+	
+CREATE INDEX old_id_index ON auditedobject USING btree (old_id, tablename);
+CREATE INDEX old_curie_index ON auditedobject USING btree (old_curie, tablename);
+
+UPDATE auditedobject t SET createdby_id = a.id FROM auditedobject a WHERE t.createdby_id = a.old_id AND a.tablename = 'person'; 
+UPDATE auditedobject t SET updatedby_id = a.id FROM auditedobject a WHERE t.updatedby_id = a.old_id AND a.tablename = 'person'; 
+
+UPDATE affectedgenomicmodel t SET id = a.id FROM auditedobject a WHERE t.curie = a.old_curie AND a.tablename = 'biologicalentity';
+UPDATE affectedgenomicmodel t SET subtype_id = a.id FROM auditedobject a WHERE t.subtype_id = a.old_id AND a.tablename = 'vocabularyterm';
+UPDATE agmdiseaseannotation t SET id = a.id FROM auditedobject a WHERE t.id = a.old_id AND a.tablename = 'association';
+UPDATE agmdiseaseannotation t SET inferredallele_id = a.id FROM auditedobject a WHERE t.inferredallele_curie = a.old_curie AND a.tablename = 'biologicalentity';
+UPDATE agmdiseaseannotation t SET assertedallele_id = a.id FROM auditedobject a WHERE t.assertedallele_curie = a.old_curie AND a.tablename = 'biologicalentity';
+UPDATE agmdiseaseannotation t SET inferredgene_id = a.id FROM auditedobject a WHERE t.inferredgene_curie = a.old_curie AND a.tablename = 'biologicalentity';
+UPDATE agmdiseaseannotation t SET subject_id = a.id FROM auditedobject a WHERE t.subject_curie = a.old_curie AND a.tablename = 'biologicalentity';
+UPDATE agmdiseaseannotation_gene t SET agmdiseaseannotation_id = a.id FROM auditedobject a WHERE t.agmdiseaseannotation_id = a.old_id AND a.tablename = 'association';
+UPDATE agmdiseaseannotation_gene t SET assertedgenes_id = a.id FROM auditedobject a WHERE t.assertedgenes_curie = a.old_curie AND a.tablename = 'biologicalentity';
+UPDATE allele t SET id = a.id FROM auditedobject a WHERE t.curie = a.old_curie AND a.tablename = 'biologicalentity';
```

Comment:
Ya so playing around with this on my system it looks like the index / pkey doesn't help:
```UPDATE allele t SET id = a.id FROM auditedobject a WHERE t.curie = a.old_curie AND a.tablename = 'biologicalentity';
-- index: 1:49
-- pkey:  1:27
-- none:  1:00
```